app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/transfers/TransfersScreenWrapper.kt

```diff
@@ -169,6 +232,7 @@ private fun NoSelectionEmptyState(hasTransfers: Boolean) {
 private data class DestinationContent(
     val direction: TransferDirection,
     val transferUuid: String,
+    val folderUuid: String? = null,
```

Comment:
This might be more of a detail because as a matter of fact this still works correctly, but wouldn't a sealed class be more appropriate here to represent the two types of navigations? If I understood correctly we might either navigate to the list of all files where `folderUuid` has no value or the a specific `folderUuid` and in this case it's set?